README.md

```diff
@@ -74,8 +74,12 @@ cd Simplified-iOS/..
 git clone https://github.com/NYPL-Simplified/r2-shared-swift
 git clone https://github.com/NYPL-Simplified/r2-streamer-swift
 git clone https://github.com/readium/r2-navigator-swift
+git clone https://github.com/NYPL-Simplified/r2-lcp-swift
 ```
-The first 2 repos contain patched versions of `R2Shared` and `R2Streamer` to enable Adobe DRM in Readium 2.
+The first 2 repos contain patched versions of `R2Shared` and `R2Streamer` to enable Adobe DRM and LCP DRM support in Readium 2.
+
+You need to obtain a link to Readium LCP binary for Carthage. 
+Once you have it, please add it to the Cartfile of SimplyE and the Cartfile of r2-lcp-swift.
```

Comment:
This module is private and build for us, and after talking to James I removed it from our Cartfile.

Reply:
who is building it? can you provide some background here? I see the repo and it seems public: https://github.com/NYPL-Simplified/r2-lcp-swift

Reply:
This is not the repo you're looking for... please check the ticket, it contains the link to the binary that's required for LCP to work

Reply:
the ticket mentions a `binary` entry for the cartfiles of r2-lcp-swift  and SimplyE, but I don't see that neither repos nor in this PR.

Reply:
Yes, that's exactly what I mean. This is a private LCP module we received from EDRL. It is not supposed to be publicly available.

Reply:
sure, that's fine, I just meant the cartfile should refer to the binary though instead of having to manually add it. Or do you mean the URL itself is not to be shared? although I do see it in https://jira.nypl.org/browse/SIMPLY-3206 which is public.

Reply:
Removed from the ticket.

Reply:
updated README, removed these lines and changed the script path to `scripts` folder